File: TZHSOJ/2023高一期末考考题/E.cpp
#include <bits/stdc++.h>
using namespace std;

signed main(){
    int a,b;
    string s;
    cin>>a>>b>>s;
    if (s=="+"){
        cout<<a+b;
    }
    else if (s=="-"){
        cout<<a-b;
    }
    else if (s=="*"){
        cout<<a*b;
    }
    else if (s=="//"){
        if (b==0){
            cout<<"Error";
            return 0;
        }
        cout<<a/b;
    }
    else if (s=="%"){
        cout<<a%b;
    }
    else {
        cout<<"Error";
    }
    return 0;
}

File: TZHSOJ/2023高一期末考考题/E.py
l=input().split()
a,b=int(l[0]),int(l[1])
c=l[2]
if(c=='+'):
    print(int(a+b))
elif(c=='-'):
    print(int(a-b))
elif(c=='*'):
    print(a*b)
elif(c=='//' and b):
    print(int(a//b))
elif(c=='%' and b):
    print(a%b)
else :
print("Error")



